File: PopAppAPI/ClientApp/src/app/models/container/container.ts
import { ContainerType } from "../../enums/container-type.enum";

export interface Container {
  containerId?: number;
  containerType?: ContainerType;
  containerPayload?: number;
  containerCapacity?: number;
  containerLenth?: number;
  containerWidth?: number;
  containerHeigth?: number;
  isActive?: boolean;
}

File: Core/Entities/Container.cs
using Core.Enums;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Core.Entities
{
    public sealed class Container
    {
        [Key]
        public int ContainerId { get; set; }
        public ContainerType ContainerType { get; set; }
        [Column(TypeName = "decimal(4,2)")]
        public decimal? ContainerPayload { get; set; }
        [Column(TypeName = "decimal(4,2)")]
        public decimal? ContainerCapacity { get; set; }
        [Column(TypeName = "decimal(4,2)")]
        public decimal? ContainerLenth { get; set; }
        [Column(TypeName = "decimal(4,2)")]
        public decimal? ContainerWidth { get; set; }
        [Column(TypeName = "decimal(4,2)")]
        public decimal? ContainerHeigth { get; set; }
        public bool IsActive { get; set; }

    }
}
